lib/Doctrine/DBAL/Driver/DriverException.php

```diff
@@ -14,29 +14,10 @@
  */
 interface DriverException extends Throwable
 {
-    /**
-     * Returns the driver specific error code if available.
-     *
-     * Returns null if no driver specific error code is available
-     * for the error raised by the driver.
-     *
-     * @return int|string|null
-     */
-    public function getErrorCode();
-
-    /**
-     * Returns the driver error message.
-     *
-     * @return string
-     */
-    public function getMessage();
-
     /**
      * Returns the SQLSTATE the driver was in at the time the error occurred.
      *
      * Returns null if the driver does not provide a SQLSTATE for the error occurred.
-     *
-     * @return string|null
      */
-    public function getSQLState();
+    public function getSQLState() : ?string;
```

Comment:
Shouldn't this BC-break be listed too?

Reply:
Why is it a BC break?

Reply:
Indeed, not a BC break: https://3v4l.org/uvKF4

Reply:
It's a BC break for those who implemented that without typehint, i.e. in mocks.

Reply:
My mistake indeed: my snippet was missing something: https://3v4l.org/4933a

Reply:
While it's a technically a BC break, it's not something worth noting in upgrade notes. Eventually, we'll have return type hints added across the entire codebase, so if documented explicitly, this list would become unreadably huge.

Reply:
That's right, a global comment on release will do